pontoon/base/templates/home.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %}
+
+{% block title %}Home Page{% endblock %}
+
+{% block home %}
```

Comment:
You should override one of the existing blocks (e.g. `middle`) instead of creating a new one. It doesn't make much sense to create page-specific blocks like `home` in a general template.

That would effectivelly mean no changes to the base template are needed.